raiden/ui/app.py

```diff
@@ -200,12 +165,14 @@ def run_app(
     if not parsed_eth_rpc_endpoint.scheme:
         eth_rpc_endpoint = f'http://{eth_rpc_endpoint}'
 
-    web3 = _setup_web3(eth_rpc_endpoint)
-    node_network_id, known_node_network_id = setup_network_id_or_exit(config, network_id, web3)
```

Comment:
`known_node_network_id` was not equal to the other two values. It was a boolean indicating whether the network is in the known networks.

Reply:
oh, thanks, I missed that. Fixing it